bigquery/samples/copy_table.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def copy_table(client, dataset_id, table_id):
+
+    # [START bigquery_copy_table]
+    # TODO(developer): Import the client library.
+    # from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    # TODO(developer): Set dataset_id to the ID of the dataset where to copy a table.
+    # dataset_id = "your-project.your_dataset"
+
+    # TODO(developer): Set table_id to the ID of the original table.
+    # table_id = "your-project.your_dataset.your_table_name"
+
+    orig_table = client.get_table(table_id)  # Make an API request.
+    dataset = client.get_dataset(dataset_id)  # Make an API request.
```

Comment:
Calls to `get_table` and `get_dataset` are unnecessary. Let's use

```
# TODO(developer): Set source_table_id to the ID of the original table.
# source_table_id = "your-project.source_dataset.source_table"

# TODO(developer): Set destination_table_id to the ID of the destination table.
destination_table_id = "your-project.destination_dataset.destination_table"
```

Reply:
I also decided to move the `num_rows` assertion to the test file.
![code_1](https://user-images.githubusercontent.com/46078689/65766189-f49c2900-e132-11e9-9e14-83254fe3b0b3.png)